src/core/wb.js

```diff
@@ -255,6 +255,16 @@ var getUrlParts = function( url ) {
 			"#999999"
 		],
 
+		// Get and generate a unique session id
```

Comment:
That's something for a separate PR. Cache busting is also better by just using the date in milliseconds